website/docs/r/vapp_container.html.markdown

```diff
@@ -0,0 +1,119 @@
+---
+layout: "vsphere"
+page_title: "VMware vSphere: vsphere_vapp_container"
+sidebar_current: "docs-vsphere-resource-compute-vapp-container"
+description: |-
+  Provides a vSphere vApp container resource. This can be used to create and manage vApp container.
+---
+
+# vsphere\_vapp\_container
+
+The `vsphere_vApp_container` resource can be used to create and manage
+Virtual Appliances.
+
+For more information on vSphere Virtual Appliances, see [this
+page][ref-vsphere-vapp].
+
+[ref-vsphere-vapp]: https://docs.vmware.com/en/VMware-vSphere/6.5/com.vmware.vsphere.vm_admin.doc/GUID-2A95EBB8-1779-40FA-B4FB-4D0845750879.html
+
+## Example Usage
```

Comment:
Do you think it's worthwhile to add another example on how to deploy a VM into a vApp container? It may not be exactly obvious from the docs to use the ID of the container as the resource pool.